nodeletctl/pkg/nodeletctl/nodeletctl.go

```diff
@@ -641,10 +626,6 @@ func SyncNodes(clusterCfg *BootstrapConfig, nodes *[]HostConfig) error {
 }
 
 func isClusterCfgValid(clusterCfg *BootstrapConfig) bool {
-	if len(clusterCfg.WorkerNodes) == 0 {
```

Comment:
Is there any scenario where this check may be valid? Do we want to allow creation of a cluster where allowWorkloadsOnMaster is false and worker nodes is zero, if not perhaps modify the check for && !clusterCfg.AllowWorkloadsOnMaster

Reply:
Good idea will fix